### PR TITLE
Documentation and Qa Testing for EdgelessDB

### PR DESCRIPTION
**Pull request by maarioos1308**
## Description

Qa testing and documentation for EdgelessDB.

**Fixes** #(#6766)

## Type of change

- [X] 📄 This change requires a documentation update

### What is the solution?

Done manual qa testing for EdgelessDB and also updated the documentation of it.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have updated the documentation, or created issues to update them.

### CODE REVIEW

File: docs/data-integrations/edgelessdb.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: EdgelessDB
+sidebarTitle: EdgelessDB
+---
+
+This is the implementation of the EdgelessDB data handler for MindsDB.
+
+[Edgeless](https://edgeless.systems/) is a full SQL database, tailor-made for confidential computing. It seamlessly integrates with your existing tools and workflows to help you unlock the full potential of your data.
+
+## Implementation
+
```
Comment (martyna-mindsdb) [inline docs/data-integrations/edgelessdb.mdx:11]: Please add this line as a first paragraph here:
```
This handler is implemented by extending the MySQL connector.
```

**Comment by martyna-mindsdb** [comment]
@maarioos1308 
Please see the review comment above - push the changes to this PR (don't create a new PR for that).

Did you test EdgelessDB in the cloud editor?

File: docs/data-integrations/edgelessdb.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* `password`: the password to authenticate the user
+* `database`: database name
+
+To use the full potensial of EdgelessDB, you can also specify the following arguments:
+
+* `ssl`: whether to use SSL or not
+* `ssl_ca`: path or url to the CA certificate
+* `ssl_cert`: path or url to the client certificate
+* `ssl_key`: path or url to the client key
+
```
Comment (martyna-mindsdb) [inline docs/data-integrations/edgelessdb.mdx:26]: Please always add this tip box at the end of Implementation section:
```
<Tip>
If you installed MindsDB locally via pip, you need to install all handler dependencies manually. To do so, go to the handler's folder (mindsdb/integrations/handlers/edgelessdb_handler) and run this command: `pip install -r requirements.txt`.
</Tip>
```

**Comment by martyna-mindsdb** [comment]
@maarioos1308
Please see the review comments above - push the changes to this PR (don't create a new PR for that).

Did you test EdgelessDB in the cloud editor?

**Comment by maarioos1308** [comment]
I didnt try that on the cloud editor because i cannot connect remotely to my local mysql database.

**Comment by martyna-mindsdb** [comment]
@maarioos1308 
You can create a tunnel and the connect your local database to MindsDB via this tunnel: https://docs.mindsdb.com/sql/create/database#making-your-local-database-available-to-mindsdb

**Comment by maarioos1308** [comment]
I have recently completed the task, and I am pleased to inform you that it is functioning perfectly. Would you like me to make any updates to the documentation?

**Comment by martyna-mindsdb** [comment]
Thanks @maarioos1308 
Glad to hear all work well in the cloud.

**Pull request merged**